### DISCUSSION TITLE
canonical timestamp

### DISCUSSION THREAD

**Discussion by truher** [robotpy/mostrobotpy#108] [General]
quick question, i'm looking for the canonical way to get the current ntcore timestamp, so i can publish the correct measurement times.  is it `ntcore._now()`?  mypy complains about touching the underscored "python pretty please private" method there.

**Comment by virtuald** [comment]
`ntcore._now` maps to `ntcore::now` (https://github.com/robotpy/mostrobotpy/blob/main/subprojects/pyntcore/gen/ntcore_cpp.yml#L16 ... https://github.com/wpilibsuite/allwpilib/blob/main/ntcore/src/main/native/include/ntcore_cpp.h#L1227)
